assets/scss/checkout.scss

```diff
@@ -1,3 +1,85 @@
 // =============================================================================
 // Stencil Checkout  - Customize the checkout experience
 // =============================================================================
+
+// Main content
+.ng-checkout-container {
```

Comment:
Are there any other classes to use for this container? `.ng-` tells me it's a class scoped for js. If there's another class to use, or a different container altogether, that'd probably be better.